### PR TITLE
Add a doc section on general usage of CPU systems

### PR DESCRIPTION
**Pull request by sbastrakov**
Solves  #4031.

@kimjohn1 @j-stephan since you recently encountered issues with setting up on CPUs, your comments and suggestions are welcome.

The section is located here: https://picongpu--4039.org.readthedocs.build/en/4039/install/profile.html#general-notes-on-using-cpus

### CODE REVIEW

File: docs/source/install/profile.rst [hunk 1]
```diff
@@ -26,10 +26,29 @@ or PIConGPU source code location defined by ``$PICSRC``.
...
 as we do not have continuous access to all of these.
 
+General Notes on Using CPUs
+---------------------------
+
+On CPU systems we strongly recommend using MPI + OpenMP parallelization.
+It requires building PIConGPU with the OpenMP 2 backend.
+Additionally it is recommended to add an option for target architecture, for example, ``pic-build -b omp2b:znver3`` for AMD Zen3 CPUs.
+When building on a compute node or a same-architecture node, one could use ``-b omp2b:native`` instead.
+The default value for ``-b`` can be set ``$PIC_BACKEND`` variable in the profile.
```
Comment (j-stephan) [inline docs/source/install/profile.rst]: This sentence sounds a bit weird.

**Review by j-stephan — COMMENTED** [review]
Thanks, this is useful! One comment:

File: docs/source/install/profile.rst (hunk 1 above)
Comment (sbastrakov) [reply docs/source/install/profile.rst]: Oh, a preposition is missing

Comment (sbastrakov) [reply docs/source/install/profile.rst]: Fixed

**Review by psychocoderHPC — APPROVED** [review]

**Pull request merged**